prompts/litellm/anthropic-claude-creative.yml

```
@@ -0,0 +1,37 @@
+  - content_type
+  - theme
+  - style
+  - audience
+references:
+  - https://docs.litellm.ai/docs/providers/anthropic
+tags:
+  - litellm
+  - anthropic
+  - claude
+  - creative
+  - writing
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)